### PR TITLE
Added google slides button

### PR DESCRIPTION
**Pull request by ericronne**
Per issue #96, i added a button linking to the google slides template,
and updated the description and subsection header (now “Presentation templates”)

### CODE REVIEW

**Comment by ericronne** [comment]
Here's the funky bullet formatting noted in the "Itemized contents of the quick-start pdf" commit. It seems that the bullets and text are being separated by a column in the grid ...

image

**Comment by ericronne** [comment]
Hi @kategarklavs! Sorry to bug you, but if you happen to have a minute, could you give a quick scan to a few changes i've made here to the branding resources page? No worries if you're over-booked—which i know you are! :( As always, please feel free to modify.

@toolness @jenniferthibault @amberwreed if you have a chance to check it out, that'd be awesome. Once Kate gives approval, perhaps one of you can do the pulling honors :)

**Comment by toolness** [comment]
I just read the markup (didn't view it locally or anything) but it all looks good to me--thanks @ericronne!

**Comment by jenniferthibault** [comment]
@toolness do you have any idea why the list items are going all formatting crazy like in Eric's screenshot above?

**Comment by toolness** [comment]
Oh sorry, I should've been paying attention to more than just the markup--one sec, lemme investigate...

File: resources.html [hunk 1]
```diff
@@ -5,16 +5,24 @@
...
-<p>Quick start reference guide for using 18F brand assets</p>
-<img src="{{ site.baseurl }}/assets/img/Instructions_Preview.png">
+<p>Quick start reference guide for using 18F brand assets, including:</p>
+<ul>
+	<li>Logo usage</li>
+	<li>How to install the brand colors on your mac</li>
+	<li>Using our brand font</li>
+	<li>Keynote theme instructions</li>
+	<li>How to use the Google Slides template</li>
+<ul> <img src="{{ site.baseurl }}/assets/img/Instructions_Preview.png">
```
Comment (toolness) [inline resources.html]: Was this meant to be a `</ul>` instead of a `<ul>`?  Changing it fixes your layout bug!

**Comment by toolness** [comment]
Ok, I think I found the problem [here](https://github.com/18F/brand/pull/97/files#r65562812)!

**Comment by ericronne** [comment]
rookie mistake
thx @toolness!

**Comment by ericronne** [comment]
note to self, don't trust random html-reference websites...
image

**Comment by ericronne** [comment]
@toolness better?

**Comment by toolness** [comment]
Yep, this looks great @ericronne !

Purely from a content perspective, I do find it a little confusing that the section title on the menu/sidebar is called "Slides", yet the corresponding heading in the content is called "Presentation templates", but I think it's probably fine.

**Comment by ericronne** [comment]
Excellent point! @jenniferthibault @amberwreed what do you think about "Slides" for nav label and "Slide templates" for the associated header?

**Comment by jenniferthibault** [comment]
I think they should be a 1:1 match exactly.  Slide templates & slide templates, or Presentation templates & Presentation templates.  (I _think_ I hear people most frequently ask for "presentation" but that's definitely not based on real analytics, just what I remember from people asking me)

**Comment by ericronne** [comment]
The site doesn't currently have a 1:1 match...
Instructions > Instructions and best practices
Keynote > Keynote template and theme

Slides > Slide templates feel dissonant?

"Slide templates" feels long, but would probably fit without wrapping

**Comment by ericronne** [comment]
@jenniferthibault all set to merge?

**Comment by jenniferthibault** [comment]
Yep, I added one commit to specify that the keynote download was a template, and not the software. Good to go otherwise, using Slides instead of Presentation.

**Pull request merged**